### ISSUE TITLE
More space for message header

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#28]
```
An individual message has an implicit "header area" with an author's name
and a time of posting. Right now these two pieces of text don't seem to sit
on a common panel and have different heights. This leads to a visual
artifact distorting flow of the message text below them when in
right-aligned layout (see screenshot attached).

The problem can be fixed by giving more vertical space to the whole header
area. This space will also make it look better when not mixed with the text.
```

Original issue reported on code.google.com by `isagal...@gmail.com` on 3 Jan 2010 at 1:29

Attachments:
- [pino-header-space.png](https://storage.googleapis.com/google-code-attachments/pino-twitter/issue-28/comment-0/pino-header-space.png)

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 3 Jan 2010 at 2:38
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Closed by b9225ab792 revision (I hope).
```

Original comment by `tro...@gmail.com` on 8 Jan 2010 at 8:19
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
The problem's gone, thanks! Though to my liking even more vertical space would 
be
better :-).
```

Original comment by `isagal...@gmail.com` on 13 Jan 2010 at 7:29
